### ISSUE TITLE
android_library doesn't recognizes CcInfo

### ISSUE THREAD

**Issue by steeve** [bazelbuild/bazel#9079]
### Description of the problem / feature request:

`android_library` doesn't recognizes `CcInfo` and complains that it only wants `cc_library`.

### Bugs: what's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.

- Return `CcInfo` from a rule
- Add that rule as a `deps` of an `android_library`

### What operating system are you running Bazel on?

macos

### What's the output of `bazel info release`?

0.28.1

### Any other information, logs, or outputs that you want to share?

The current work around is to create an intermediate `cc_library`
```python
cc_library(
    name = cc_library_name,
    deps = [rule_that_returns_cc_info],
)
android_library(
    name = android_library_name,
    deps = [cc_library_name],
)
```

I do think that `java_library` properly consumes `CcInfo`, however.

**Comment by gyias** [comment]
This is fixed now, with 1.2 release. Closing.

**Comment by steeve** [comment]
Awesome thank you!
